packages/plugins/apollo-usage-report/src/index.ts

```diff
@@ -54,6 +57,11 @@ function getEnvVar<T>(name: string, defaultValue?: T) {
   return globalThis.process?.env?.[name] || defaultValue || undefined;
 }
 
+const reportHeaderDefaults: IReportHeader = {
+  agentVersion: `${packageJson.name}@${packageJson.version}`,
+  runtimeVersion: `node ${process.version}`,
```

Comment:
This is not always true, Yoga is runtime agnostic, which means it doesn't run only in Node, it can be Deno, Bun, or any JS Edge runtime.